src/components/SmartInput/CustomSelectComponents/CustomSelectedValue.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { getValueType } from '../utils';
+import { OptionProps } from '..';
+import { Props } from 'react-select';
+
+export interface SelectedValueProps {
+  item: OptionProps;
+  selectProps: Props<OptionProps, true>;
+}
+
+export const SelectedValue = ({ item, selectProps }: SelectedValueProps): JSX.Element => {
+  const { text, defaultType, isExpert, type } = item;
+  const { setCursorPosition, value, onSetSearchItems } = selectProps;
+  const itemIndex = Array.isArray(value) ? value?.findIndex((i) => i === item) : 0;
```

Comment:
`const itemIndex = Array.isArray(value) ? value.findIndex((i) => i === item) : 0;`

isArray would return false if the value it is null or undefined so it is not need to use the optional operator

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#using_array.isarray